config/locales/views/talk/show/en.yml

```diff
@@ -0,0 +1,7 @@
+# See https://github.com/svenfuchs/rails-i18n/tree/master/rails%2Flocale for starting points.
+
+en:
+  talk:
+    show:
+      welcome: "Hello Everyone! Welcome to Publiclab Talk Pad"
```

Comment:
yes! Sorry, just got the email ping. Suggestion below in pink::
Welcome to the Public Lab Talk Pad

This page allows for synchronized editing by multiple people as you type.
It can be used to help you:
- collaborate with fellow wiki editors,
- take notes with multiple people in real time, and
- log the changes, decisions and notes made about a wiki page, or a topic.
Etherpad Lite on Github: http://j.mp/ep-lite


Open to suggestions!
-Stevie

On Tue, Aug 9, 2016 at 5:40 AM, Jeffrey Warren <notifications@github.com>
wrote:

> In config/locales/views/talk/show/en.yml
> <https://github.com/publiclab/plots2/pull/645#discussion_r74035107>:
>
> > @@ -0,0 +1,7 @@
> > +# See https://github.com/svenfuchs/rails-i18n/tree/master/rails%2Flocale for starting points.
> > +
> > +en:
> > +  talk:
> > +    show:
> > +      welcome: "Hello Everyone! Welcome to Publiclab Talk Pad"
>
> Hi, @jiteshjha <https://github.com/jiteshjha> - if you could ask stevie
> and liz (each @publiclab.org) via email about this, that'd be great.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/publiclab/plots2/pull/645/files/d12cbef0d1f61cd7356075bd8797ca8f0831107d#r74035107>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHJLrh7zfmPr8jUC90moIdh_S9n8bMxdks5qeFkDgaJpZM4JZK7K>
> .
>